src/main/java/pl/craftserve/radiation/LugolsIodinePotion.java

```diff
@@ -309,5 +337,56 @@ public Duration duration() {
         public Optional<String> drinkMessage() {
             return Optional.ofNullable(this.drinkMessage);
         }
+
+        public Recipe getRecipe() {
+            return recipe;
+        }
+
+        public static class Recipe {
+            private final boolean enabled;
+            private final PotionType basePotion;
+            private final Material ingredient;
+
+            public Recipe(boolean enabled, Material ingredient, PotionType basePotion) {
+                this.enabled = enabled;
+                this.ingredient = Objects.requireNonNull(ingredient, "ingredient");
+                this.basePotion = Objects.requireNonNull(basePotion, "basePotion");
+            }
+
+            public Recipe(ConfigurationSection section) throws InvalidConfigurationException {
+                if (section == null) {
+                    section = new MemoryConfiguration();
+                }
+
+                this.enabled = section.getBoolean("enabled", true);
+                if (this.enabled) {
+                    this.ingredient = Material.matchMaterial(Objects.requireNonNull(section.getString("ingredient", "ghast_tear")));
```

Comment:
```suggestion
                    this.ingredient = Material.matchMaterial(Objects.requireNonNull(section.getString("ingredient", Material.GHAST_TEAR.getKey​().getKey())));
```

The compiler will then fail if they change the material, instead of trying to load default options on the server.